src/helpers.js

```diff
@@ -86,6 +86,17 @@ export async function normalizeConfig(givenConfig: ConfigGiven): Promise<Config>
       throw new Error('config.password must be a string')
     }
   }
+
+  if (config.tryKeyboard === true && typeof config.onKeyboardInteractive !== 'function') {
```

Comment:
Could we also set the default value of `tryKeyboard` here somewhere?

Reply:
Sure!